src/go/configmanager/flags/flags.go

```diff
@@ -57,17 +57,14 @@ var (
 	ListenerPort = flag.Int("listener_port", 8080, "listener port")
 	Healthz      = flag.String("healthz", "", "path for health check of ESPv2 proxy itself")
 
+	ServerSslPath = flag.String("server_ssl_path", "", "Path to the certificate and key that ESPv2 uses to act as a HTTPS server")
```

Comment:
how ssl_server_path?   we can prefix all ssl related flags with ssl_

Reply:
I feel server_ssl_path/client_ssl_path is better than ssl_server_path/ssl_client_path.

Reply:
I feel that if all ssl related flags can be prefixed with "ssl_",  it will be easier for users to find it.

Reply:
Done, changed to ssl_server_path